src/main/java/com/puppycrawl/tools/checkstyle/checks/header/RegexpHeaderCheck.java

```diff
@@ -151,9 +167,15 @@ private boolean isMultiLine(int lineNo) {
     @Override
     protected void postProcessHeaderLines() {
         final List<String> headerLines = getHeaderLines();
-        for (String line : headerLines) {
+        final Pattern blankLine = Pattern.compile(EMPTY_LINE_PATTERN);
```

Comment:
please move compilation of pattern to class field, to be done ones, for whole execution of checkstyle.

Reply:
Done